packages/create-cloudflare/src/helpers/args.ts

```diff
@@ -39,6 +39,10 @@ export const parseArgs = async (argv: string[]): Promise<Partial<C3Args>> => {
 			alias: "y",
 			type: "boolean",
 		})
+		.option("overwrite", {
+			type: "boolean",
+			description: "Overwrite files if target directory exists.",
```

Comment:
```suggestion
			description: "Overwrite files if target directory exists",
```
(the other descriptions don't end with a full stop)